gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -99,6 +101,14 @@
               </div>
               <p translate class="help-block">File permission bits are ignored when looking for changes. Use on FAT file systems.</p>
             </div>
+            <div ng-if="currentFolder.type == 'receiveonly'" class="form-group">
+              <div class="checkbox">
+                <label>
+                  <input type="checkbox" ng-model="currentFolder.deleteLocalChanges"><span translate>Delete local changes</span>
```

Comment:
I first wrote this comment:

> I don't think "delete" is the right word here, in the description and config name - isn't "revert" more what we're doing? (i.e., we can't delete a change that deletes a file, but we can revert it)

then later realized that this is because it always reverts something that is not an addition, hence this only controls deletes of additions. That is not in the spirit of how I want things to work in order to minimize damage. I would expect this to be *be* a true toggle of `revertLocalChanges` so that when it's false we do not stomp on the user's changes - we just don't send them onward towards the cluster.